src/components/CircleImage/CircleImage.js

```diff
@@ -25,93 +25,93 @@ function sumChars(str) {
   return sum;
 }
 
-export default class CircleImage extends Component {
-  static propTypes = {
-    borderRadius: PropTypes.string,
-    src: PropTypes.string,
-    srcset: PropTypes.string,
-    name: PropTypes.string.isRequired,
-    color: PropTypes.string,
-    colors: PropTypes.array,
-    size: PropTypes.string,
-    style: PropTypes.object,
-    className: PropTypes.string,
-  };
+const CircleImage = props => {
+  const {
+    src,
+    srcset,
+    name,
+    color,
+    colors,
+    style,
+    borderRadius = '100%',
+    className,
+  } = props;
 
-  static defaultProps = {
-    colors: defaultColors,
-  };
+  let size = addPx(props.size);
 
-  render() {
-    const {
-      src,
-      srcset,
-      name,
-      color,
-      colors,
-      style,
-      borderRadius = '100%',
-      className,
-    } = this.props;
+  let styles = {
+    imageStyle: {
+      display: 'block',
+      borderRadius,
+    },
+    innerStyle: {
+      lineHeight: size,
+      textAlign: 'center',
+      overflow: 'initial',
+      marginRight: 10,
+      borderRadius,
+    },
+  };
 
-    let size = addPx(this.props.size);
+  let { imageStyle, innerStyle } = styles;
+  const nameInitials = initials(name);
+  let innerElement,
+    classes = [className, 'CircleImage'];
```

Comment:
classes could be declared as const here

Reply:
They are being reassigned, so let it be.

Reply:
Oh right. Completely missed that. Sorry.